CONTRIBUTING.md

```diff
@@ -61,6 +64,34 @@ Every new file must contain the Eclipse license information and the
 copyright holder(s). Please take a look into existing files and adopt
 the needed changes to your new file(s).
 
+Submit Patches via Github:
+--------------------------
+
+For patches via github PullRequests, the same rules as via gerrit applies.
+
+Main and Develop:
+-----------------
+
+Please prepare all patches against the "main" branch.
+
+It may take sometimes a little longer for a pull request to be processed
+and merged to "main". Therefore some of the pending PRs will be available
+on the "develop" branch as preview. If you want to test a specific pending
+pull request, which is currently not on "develop", let us know by adding
```

Comment:
No comma before "which"  (keep the other one for clarity).

Reply:
OK